vc_zoom/tests/operation_test.py

```diff
@@ -107,6 +107,7 @@ def test_room_creation(create_meeting, zoom_api):
 
 
 def test_host_change(create_user, mocker, create_meeting, zoom_plugin, zoom_api, request_context):
+    mocker.patch('indico_vc_zoom.plugin.find_enterprise_email', return_value='joe.bidon@megacorp.xyz')
```

Comment:
This kind of keeps `find_enterprise_email` from being covered. Maybe we should add a separate test for it in that case?

Reply:
Good point, I'll add one later. Not super straightforward to write a test for it because of all the dependencies that need to be mocked out.